### PR TITLE
Add new resource console_plugin

### PR DESCRIPTION
**Pull request by dbasunag**

### CODE REVIEW

**Comment by dbasunag** [comment]
!verified

File: ocp_resources/console_plugin.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import Resource
+
+
+class ConsolePlugin(Resource):
+    """
+    ConsolePlugin object.
+    """
+
+    api_group = Resource.ApiGroup.CONSOLE_OPENSHIFT_IO
+    api_version = Resource.ApiVersion.V1ALPHA1
```
Comment (myakove) [inline ocp_resources/console_plugin.py:10]: api_version should be set automatically.

**Review by myakove — CHANGES_REQUESTED** [review]

File: ocp_resources/console_plugin.py (hunk 1 above)
Comment (dbasunag) [reply ocp_resources/console_plugin.py:10]: I will remove this.

**Comment by dbasunag** [comment]
!verified

**Review by myakove — APPROVED** [review]

**Pull request merged**